### PR TITLE
Update directory components

### PR DESCRIPTION
**Pull request by jessgilbert**
To do (delete all that do not apply):

 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled

### CODE REVIEW

**Comment by MichaelWalker** [comment]
I _think_ we'll also need to kill the `dit.tagging.international` file and tweak base.html to reflect that.

**Comment by MichaelWalker** [comment]
also this file has been killed, so we should delete it from base.html to fix the tests - `Missing staticfiles manifest entry for 'directory_components/js/dit.tagging.internationalHeader.js'`

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,21 @@
 
 ## Pre-release
 
+### Fixed bugs:
+- No ticket - Upgrade vulnerable django version to django 1.11.22
```
Comment (MichaelWalker) [inline CHANGELOG.md]: I thought that this had already been done?

Comment (jessgilbert) [reply CHANGELOG.md]: Ah. Changelog has been updated and merged since the release so things may now be out of place

**Pull request merged**